src/applications/claims-status/components/appeals-v2/DurationCard.jsx

```diff
@@ -10,16 +10,22 @@ const DurationCard = ({ durationText, cardDescription, isDisabled = true }) => {
   }
 
   return (
-    <div className="card information">
-      <span className="number">{durationText}</span>
-      <span className="description">{cardDescription}</span>
-    </div>
+    <va-card background class="duration-card">
+      <div className="number vads-u-font-family--serif 40px vads-u-font-size--h2 vads-u-font-weight--bold vads-u-line-height--1 vads-u-padding-bottom--2 vads-u-border-bottom--1px vads-u-border-color--gray-light">
+        <i className="fas fa-clock vads-u-margin-right--1p5" alt="clock icon" />
+        {durationText}
+      </div>
+      <div className="description vads-u-font-size--base vads-u-font-weight--normal vads-u-padding-top--0p5">
+        {cardDescription}
+      </div>
+    </va-card>
```

Comment:
This looks like a lot of changes but I just transfered the [deprecated .card Formation styles](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/f78cc125b45f07273409046fea7c4e21af71f46d/packages/formation/sass/base/_va.scss#L656-L736) into [Design System utility classes](https://design.va.gov/foundation/utilities/). See the PR description for a screenshot.